src/components/Nav.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+const Nav = () => {
+  const [navOpen, setNavOpen] = useState(false);
+  const links = [
+    { title: "Home", link: "/" },
+    { title: "About Us", link: "/about" },
+    { title: "Map", link: "/map" },
+    { title: "Methods", link: "/methods" },
+    { title: "Further Readings", link: "/further-reading" },
+    { title: "How To Help", link: "/how-to-help" },
+  ];
+  function toggle() {
+    setNavOpen((prevNavOpen) => !prevNavOpen);
+  }
+
+  return (
+    <div className="nav-header-bar">
+      <div className={`nav-blur-overlay ${navOpen ? "open" : ""}`}></div>
```

Comment:
Thoughts on removing this overlay? I'm not sure how I feel about the blur when the navigation appears, but if you feel strongly I could be convinced